src/clang_parser.cpp

```diff
@@ -633,11 +661,40 @@ bool ClangParser::parse(ast::Program *program, BPFtrace &bpftrace, std::vector<s
     return false;
   }
 
-  if (!handler.check_diagnostics(input))
+  std::vector<std::string> error_msgs;
+  if (!handler.check_diagnostics(input, error_msgs, true))
+  {
+    for (auto &msg : error_msgs)
+    {
+      if (get_unknown_type(msg) != "")
```

Comment:
```suggestion
      if (get_unknown_type(msg) != "" && !bpftrace.force_btf_)
```
just to avoid a redundant message (if we leave `force_btf_` option.)